lib/core/utils.js

```diff
@@ -448,18 +448,8 @@ RESUtils.currentSubreddit = function(check) {
 	}
 };
 RESUtils.thingsContainer = function(body) {
-	var container;
 	body = body || document.body;
-	container = body.querySelectorAll(RESUtils.thing.prototype.containerSelector);
-	// stupid sponsored links create a second div with ID of sitetable (bad reddit! you should never have 2 IDs with the same name! naughty, naughty reddit!)
-	if (container.length === 2) {
-		// console.log('skipped first sitetable, stupid reddit.');
-		container = container[1];
-	} else {
-		container = container[0];
-	}
-
-	return container;
+	return $(body).find(RESUtils.thing.prototype.containerSelector)[0];
```

Comment:
Er, nothing else uses this does it?
I don't think we can really do this with vanilla selectors, unless we do `selected[selected.length - 1]`